Packages/vcs/vcs/Canvas.py

```diff
@@ -4961,6 +4979,15 @@ def setcolormap(self, name):
         #   updateVCSsegments_flag = args[1]
         # except:
         #   updateVCSsegments_flag = 1
+        # Python < 3 DeprecationWarning ignored by default
```

Comment:
> remember there are 3 levels to look for the colormap used.
> 1- graphics method
> 2- canvas
> 3 vcs module

Okay, let me add something to the code to make sure that we can map old to new regardless of where we set the colormap.